srv/modules/runners/validate.py

```diff
@@ -593,21 +616,25 @@ def pillars(**kwargs):
     #options = SaltOptions()
     local = salt.client.LocalClient()
     cluster = ClusterAssignment(local)
+    
+    printer = printer = get_printer(**kwargs)
+
 
     for name in cluster.names:
-        pillar(name, **kwargs)
+        pillar(name, printer=printer, **kwargs)
+    
+    printer.print_result()
 
 
-def pillar(name = None, **kwargs):
+def pillar(name = None, printer=None, **kwargs):
     """
     Check that the pillar for each cluster meets the requirements to install
     a Ceph cluster.
     """
-
-    if name == None:
-        if 'name' in kwargs:
-            name = kwargs['name']
-
```

Comment:
You wouldn't have both... More an exclusive OR usage.  At the command line, most salt commands seem to allow/encourage plain args such as

salt-run validate.pillar ceph

but when I call that in an sls file, the kwargs syntax is a bit more informative and the only method I could get working for runners under the reactor.  

add minion:
  runner.filequeue.enqueue:
    - kwargs:
        queue: prep 
        item: {{ data['id'] }}

with the two different usages, that seemed a reasonable option.